packages/manager/apps/public-cloud/src/components/beta-warning/index.js

```diff
@@ -0,0 +1,19 @@
+import angular from 'angular';
+import 'ovh-ui-angular';
+import ngOvhSidebarMenu from '@ovh-ux/ng-ovh-sidebar-menu';
+
+import component from './beta-warning.component';
+
+import './beta-warning.less';
+
+const moduleName = 'ovh-manager-betaWarning';
```

Comment:
```suggestion
const moduleName = 'ovhManagerBetaWarning';
```
to keep consistency with other namings